src/request/reducers/RequestDetailDataReducer.ts

```diff
@@ -134,6 +142,8 @@ function createMongoDbDeleteOperation(message: IMessageEnvelope<IDataMongoDbDele
     return {
         id: message.id,
         database: 'MongoDB',
+        databaseName: message.payload.database,
+        serverName: message.payload.connectionHost,
```

Comment:
Wonder if its worth creating a `payload` const and stashing the data there... might save a few bytes. That said, when the product pack is done, it might optimize for cases like this.